woocommerce/woocommerce
# Skip e2e tests for not relevant changes (e.g. docs)

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #44328 .

There is no need to run time consuming e2e tests when there's only a documentation update.
Added a path-ignore for all md files for `Run tests against PR`, `Run tests against PR in an environment with HPOS disabled` and `Metrics Tracking` workflows.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create another PR from a branch having this one as base.
2. Update any `.md` file
3. Check the CI. The e2e tests or the performance tests should not run.
Tests with #44317 



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
